### ISSUE TITLE
Media Playback Quality

### ISSUE THREAD

**Issue by padenot** [mozilla/standards-positions#185]
## Request for Mozilla Position on an Emerging Web Specification

* Specification Title: Media Playback Quality
* Specification or proposal URL: https://wicg.github.io/media-playback-quality/
* Caniuse.com URL (optional): Not found
* Bugzilla URL (optional): https://bugzilla.mozilla.org/show_bug.cgi?id=889205
* Mozillians who can provide input (optional): @jyavenard @kinetik

### Other information

Firefox has shipped this for some time.

**Comment by marcoscaceres** [comment]
As with the others, as we've shipped, I guess we consider this "important"?

**Comment by padenot** [comment]
Yes, this is also being used by major websites to provide the best content to their viewers, this is being relied on.
